packages/fxa-settings/src/models/NonSettingsContext.ts

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { mockNonSettingsContext } from './mocks';
+
+export interface NonSettingsContextValue {
+  integration: Integration;
+}
+
+// TODO: account for 'pairing' auth-brokers
+export enum IntegrationType {
+  OAuthRedirect,
+  OAuthWebChannel,
+  Sync,
+  SyncChannel,
+  SyncWebChannel,
+  V3Desktop,
+  Web,
+}
+
+type Integration = {
+  type: IntegrationType;
+  features: Partial<IntegrationFeatures>;
+};
+
+type IntegrationFeatures = {
+  allowUidChange: boolean;
+  browserTransitionsAfterEmailVerification: boolean;
```

Comment:
I think we can remove these feature integrations, browserTransitionsAfterEmailVerification, tokenCode

Reply:
Ah good catch. `browserTransitionsAfterEmailVerification` is only used in fx-ios-v1 and `tokenCode` is never set to false. 😁